Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -2280,7 +2280,9 @@ public static partial class PythonOps {
         public static PythonDictionary CopyAndVerifyDictionary(PythonFunction function, IDictionary dict) {
             foreach (object? o in dict.Keys) {
                 if (!(o is string)) {
-                    throw TypeError("{0}() keywords must be strings", function.__name__);
+                    if (!(o is Extensible<string> es)) {
```

Comment:
Maybe combine the ifs?

Reply:
Sure, the same in `UserMappingToPythonDictionary` I presume.